app/src/main/java/com/olayg/shibeonline/adapter/ShibeAdapter.kt

```diff
@@ -36,6 +37,7 @@ class ShibeAdapter : RecyclerView.Adapter<ShibeAdapter.ShibeViewHolder>() {
 
         fun loadUrl(url: String) = with(binding) {
             //ivImage.load(url)
+            Glide.with(root).load(url).into(ivImage)
```

Comment:
Create an extension function on the `ImageView` Class this way it has the ability to load urls